README.md

```diff
@@ -2246,6 +2261,46 @@ simdutf_warn_unused size_t maximal_binary_length_from_base64(const char * input,
  */
 simdutf_warn_unused size_t maximal_binary_length_from_base64(const char16_t * input, size_t length) noexcept;
 
+/**
+ * Provide the exact binary length in bytes given the base64 input.
```

Comment:
Let's not overpromise here, making an API that is harder to accelerate.

The original issue only promised the correct answer for well-formed base64 with optional ASCII spaces. It doesn't make any promises about only counting actual base64 characters.